src/main/java/de/srendi/advancedperipherals/common/addons/refinedstorage/RsStorageTypes.java

```diff
@@ -0,0 +1,25 @@
+package de.srendi.advancedperipherals.common.addons.refinedstorage;
+
+import com.refinedmods.refinedstorage.common.api.storage.StorageType;
+import com.refinedmods.refinedstorage.common.storage.StorageTypes;
+import com.refinedmods.refinedstorage.mekanism.ChemicalResourceType;
+
+/**
+ * To better support third party RS addons and to prevent any jvm loading issues when third party addons are not loaded
+ */
+public enum RsStorageTypes {
+
+    ITEM(StorageTypes.ITEM),
+    FLUID(StorageTypes.FLUID),
+    CHEMICAL(ChemicalResourceType.STORAGE_TYPE);
```

Comment:
if mekanism is not loaded, can you use `ChemicalResourceType`?  It will be strange if `RsStorageTypes` can be success initialized.

Reply:
yeah it does not, not sure why I didn't thought about that.
But I am at a point where I am not sure how to handle that properly without spaghetti

Reply:
![image](https://github.com/user-attachments/assets/c6c7b8a4-2580-4e09-871f-f0efbc5affad)

yeah fixed it
No exceptions, no crashes